atomic_reactor/omps_util.py

```diff
@@ -0,0 +1,95 @@
+"""
+Copyright (c) 2019 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
```

Comment:
Should this module be implemented in OMPS upstream and imported here?

Reply:
I'm using OSBS specific retries. I'd not complicate things with 3rd party dependencies. OMPS support only py3.6+